docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes.md

```diff
@@ -60,15 +60,107 @@ showAsideToc: true
 
 ![Configure Cloud Provider](/images/ee/configure-cloud-provider.png)
 
 ## Prerequisites
 
 ### Kubernetes
 
-If you plan to run YugabyteDB nodes on Kubernetes, all you need to provide in the Yugabyte Platform console is your cloud provider credentials. The Yugabyte Platform will use those credentials to automatically provision and de-provision instances that run Yugabyte. An 'instance' for YugabyteDB includes a compute instance as well as local or remote disk storage attached to the compute instance.
+If you plan to run YugabyteDB nodes on Kubernetes, all you need to provide in the Yugabyte Platform console is your Kubernetes provider credentials. The Yugabyte Platform will use those credentials to automatically provision and de-provision the pods that run Yugabyte.
+
+Before you install YugabyteDB on a Kubernetes cluster, perform the following:
+
+- Create a `yugabyte-platform-universe-management` service account.
+- Create a `kubeconfig` file of the earlier created service account to configure access to the Kubernetes cluster.
+
+### Service account creation
+
+This is the ServiceAccount whose secret can be used to generate a kubeconfig.
+
+**Notes**
+
+- It should not be deleted once it is being used by the platform.
+- `kube-system` namespace in the ServiceAccount creation command can be replaced by the desired namespace in which to install YugabyteDB.
+
+Run the following `kubectl` command to apply the YAML file:
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/yugabyte/charts/master/rbac/yugabyte-platform-universe-management-sa.yaml -n kube-system
+```
+
+The following output should appear:
+
+```
+serviceaccount/yugabyte-platform-universe-management created
+```
+
+The next step is to grant access to this ServiceAccount using ClusterRoles/Roles and ClusterRoelBindings/RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you.
+Follow any one of the following steps depending on your requirements.
+
+**Notes**
+- Make sure you replace the [namespace] from the commands with the correct namespace of the earlier created ServiceAccount.
+
+**Global Admin**
+
+Grants broad cluster level admin access.
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-global-admin.yaml -n <namespace>
+```
+
+**Global Restricted**
+
+Grants access to only the specific cluster roles to create and manage YugabyteDB universes across all the namespaces in a cluster. Contains ClusterRoles and ClusterRoleBindings for the required set of permissions.
+
+```sh
+kubectl apply -f https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-global.yaml -n <namespace>
+```
+
+**Namespace Admin**
+
+Grants namespace level admin access.
+
+If you have multiple target namespaces, then you will have to apply the YAML in all the target namespaces.
```

Comment:
then you will have to apply the YAML in all the target namespaces. --> then you have to apply the YAML in all of them.